src/ui/components/Scanner/Scanner.tsx

```diff
@@ -147,6 +157,7 @@ const Scanner = forwardRef(
 
         if (invitation.type === KeriConnectionType.MULTI_SIG_INITIATOR) {
           setGroupId(invitation.groupId);
+          handleMultisigConnections(invitation.groupId);
```

Comment:
This only covers the case where you scan the initiator of the multi-sig. It doesn't cover the case where the initiator scans the other person, or if there are more than 2 participants and they scan each other. You'll need to cover this around line 153 too.

Reply:
In that case, should I use the `connections `when call `getMultisigLinkedContacts `to add to the store?